### PR TITLE
Use canary tag for k8sdb images

### PR DESCRIPTION
**Pull request by aerokite**
Using canary tag for k8sdb images for examples

### CODE REVIEW

File: pkg/controller/database_snapshot.go [hunk 1]
```diff
@@ -15,11 +15,11 @@ import (
 )
 
 const (
-	imageElasticDump        = "appscode/elasticdump"
+	imageElasticDump        = "k8sdb/elasticdump"
 	SnapshotProcess_Backup  = "backup"
 	snapshotType_DumpBackup = "dump-backup"
 	storageSecretMountPath  = "/var/credentials/"
-	tagElasticDump          = "2.4.2-v2"
+	tagElasticDump          = "canary"
```
Comment (TamalSaha) [inline pkg/controller/database_snapshot.go]: This should be flag.

File: pkg/controller/create.go [hunk 2]
```diff
@@ -17,9 +17,9 @@ import (
 const (
 	annotationDatabaseVersion  = "elastic.k8sdb.com/version"
 	GoverningElasticsearch     = "governing-elasticsearch"
-	imageElasticsearch         = "appscode/elasticsearch"
-	imageOperatorElasticsearch = "appscode/k8s-es"
-	tagOperatorElasticsearch   = "0.1"
+	imageElasticsearch         = "k8sdb/elasticsearch"
+	imageOperatorElasticsearch = "k8sdb/k8s-es"
+	tagOperatorElasticsearch   = "canary"
```
Comment (TamalSaha) [inline pkg/controller/create.go]: This should be a flag.

File: pkg/controller/database_snapshot.go (hunk 1 above)
Comment (aerokite) [inline pkg/controller/database_snapshot.go]: done

File: pkg/controller/create.go (hunk 2 above)
Comment (aerokite) [inline pkg/controller/create.go]: done

**Pull request merged**